composer.json

```diff
@@ -14,6 +14,12 @@
       "email": "dominik@pfaffenbauer.at",
       "homepage": "https://www.pfaffenbauer.at/",
       "role": "Developer"
+    },
+    {
+      "name": "Korneliusz Kirsz",
```

Comment:
not sure about that ;)

I think we should add the community/contributors with one entry: https://github.com/symfony/symfony/blob/master/composer.json#L14
https://github.com/Sylius/Sylius/blob/master/composer.json#L17

Reply:
done